Resources/public/javascript/cropper.js

```diff
@@ -0,0 +1,24 @@
+class Cropper {
+    constructor(item) {
+        this.item = item;
+        this.item.find($('*[data-method="getCroppedCanvas"]')).on('click', this.save.bind(this));
+    } 
+
+    save() {
+        let updateBtn = $('*[name="btn_update_and_edit"]');
+        if(!updateBtn.length) return;
```

Comment:
So the only function this class has is to save, but why should the class exist in the first place if there is not save button available?